todo.js

```diff
@@ -15,3 +15,32 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+let process = require('process');
+let fs = require('fs');
+// let readlineSync = require('readline-sync');
+
+function listTasks() {
```

Comment:
Tightly coupling the input and display concerns works fine for these two scenarios, but it's going to be annoying if you ever want to...

1. Edit a todo item
2. Delete a todo item
3. Change the order in which the todos are printed without changing how they're stored on disk

Model the todo list in code first, in a way that's agnostic to where the data comes from and how it's displayed. Write code that ties user input to the manipulation of in-memory objects.

Reply:
One way you can tell that this function does way too much: try to write sanity checks for it.